### PR TITLE
Refactor unit tests

### LINKED ISSUE #759: Refactor unit tests
### Description

Refactor all the unit test cases to use `testing` interface exposed from SDK. All the scenarios mention [here](https://docs.google.com/spreadsheets/d/1aVYgWw8Fn___tq7nOy_vSo2w3UUvaOsvHy1wWAGLhAY/edit#gid=0) should be covered.

### Motivation

Current implementation of unit tests does not cover all the scenarios. For proper testing, we can use `testing` interface from SDK.

### Additional Information
Ref: https://docs.google.com/spreadsheets/d/1aVYgWw8Fn___tq7nOy_vSo2w3UUvaOsvHy1wWAGLhAY/edit#gid=0
Ref: https://github.com/LiskHQ/lisk-sdk/blob/development/framework/test/unit/modules/pos/commands/validator_registration.spec.ts

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?

This PR resolves #759 

### How was it solved?
- [x] Update test coverage for reclaim transaction
- [x] Update test coverage for registerBLSKey transaction
### How was it tested?
 - [x] Local

### CODE REVIEW

File: test/application/modules/legacy/commands/reclaim.spec.ts [hunk 1]
```diff
@@ -12,64 +12,102 @@
...
+	VerifyStatus,
+	testing,
+	codec,
+	Transaction,
+	cryptography,
+	EventQueuer,
+} from 'lisk-sdk';
+
+// TODO: Update this once exposed from SDK
+import { PrefixedStateReadWriter } from '../../../../../node_modules/lisk-framework/dist-node/state_machine/prefixed_state_read_writer';
```
Comment (sameersubudhi) [inline test/application/modules/legacy/commands/reclaim.spec.ts:26]: Please create an issue in the SDK repo for the same.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: test/application/modules/legacy/commands/reclaim.spec.ts [hunk 2]
```diff
@@ -87,75 +125,85 @@ describe('Reclaim command', () => {
...
-				eventQueue,
-			);
-
-			when(mockStoreHas).calledWith(legacyAddress).mockReturnValue(true);
-			when(mockGetWithSchema)
-				.calledWith(legacyAddress, legacyAccountResponseSchema)
-				.mockReturnValue({ balance: reclaimBalance });
-
-			await expect(reclaimLSKCommand.verify(commandVerifyContextInput)).resolves.toHaveProperty(
+		it(`should return status when called with valid input`, async () => {
```
Comment (nagdahimanshu) [inline test/application/modules/legacy/commands/reclaim.spec.ts:128]:
```suggestion
		it('should return status when called with valid input', async () => {
```

**Review by nagdahimanshu — APPROVED** [review]
Looks good apart from one minor comment 👍

File: test/application/modules/legacy/commands/reclaim.spec.ts (hunk 1 above)
Comment (priojeetpriyom) [reply test/application/modules/legacy/commands/reclaim.spec.ts:26]: https://github.com/LiskHQ/lisk-sdk/issues/8372

File: test/application/modules/legacy/commands/reclaim.spec.ts (hunk 2 above)
Comment (priojeetpriyom) [reply test/application/modules/legacy/commands/reclaim.spec.ts]: Applied.

**Review by nagdahimanshu — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**